mavlink_vehicles/mavlink_vehicles.cc

```diff
@@ -41,6 +41,38 @@
 #define print_verbose(...) ;
 #endif
 
+enum PX4_CUSTOM_MAIN_MODE {
```

Comment:
I think I will create two new headers then: px4.hh and apm.hh. The first will have these definitions and other px4 specific constants while in the second I will keep the the ardupilot constants, as [this](https://github.com/01org/mavlink-vehicles/pull/28/files#r83846026) one. Currently we're selecting the autopilot in compile time, but we want to detect it in runtime in the future. I will send a separate patch with that feature.